Source/WebCore/html/canvas/WebGLVertexArrayObject.cpp

```diff
@@ -51,10 +51,6 @@ WebGLVertexArrayObject::~WebGLVertexArrayObject()
 WebGLVertexArrayObject::WebGLVertexArrayObject(WebGLRenderingContextBase& context, Type type)
     : WebGLVertexArrayObjectBase(context, type)
 {
-#if USE(OPENGL_ES)
```

Comment:
Shouldn't we remove the remove the #if check here, but keep the code inside as we're using GLES?